tests/test_smart_ptr.py

```diff
@@ -1,125 +0,0 @@
-from pybind11_tests import ConstructorStats
```

Comment:
You can also tell pytest to skip an entire .py file by adding the following global variable to it:
```python
pytestmark = pytest.mark.skip(reason="why")
```